### PR TITLE
feat: Rating and grid added

### PR DESCRIPTION
**Pull request by sumitch29**
- Rating and grid added
- added favicon
- added information in some component
- added top icons of GitHub, LinkedIn, Twitter
- added get started page
Screenshot (374)
Screenshot (375)
Screenshot (376)
Screenshot (377)
Screenshot (378)
Screenshot (379)
Screenshot (380)

### CODE REVIEW

**Review by Pratham82 — COMMENTED** [review]
Good to go, just add some padding to the content in the grid so it would be better to differentiate the info.

File: Components/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    <a class="sidebarlink" href="/Components/Grid/grid.html">Grid</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="page-container">
+            <h1 class="heading">Rating</h1>
+            <p class="info">Rating component is used to give stars rating for a product reviews</p>
+            <h1></h1>
+            <div class="box">
+                <div class="star-rating">
```
Comment (krgarima) [inline Components/Rating/rating.html:70]: Just a simple suggestion: You can mention the number of people who rated for the product. Example: 12,833 ratings.

**Review by krgarima — COMMENTED** [review]
Website looks awesome! Good to go.

**Review by krgarima — COMMENTED** [review]
Website looks good! Good to go.

**Pull request merged**